server/src/views/frame.js

```diff
@@ -563,9 +563,6 @@ class Frame extends React.Component {
         { this.props.shot.showPage ? <span id="copy-flag">Copy</span> : null }
         { this.props.shot.showPage ?
           <iframe width="100%" height={frameHeight} id="frame" src={ shot.contentUrl } style={ {backgroundColor: "#fff"} } /> : null }
-        <div className="pageshot-footer">
-          <a href="https://github.com/mozilla-services/pageshot" onClick={ this.onClickGitHub.bind(this) }>{this.props.productName}</a> — <a href={`https://github.com/mozilla-services/pageshot/commit/${getGitRevision()}`} onClick={ this.onClickRevision.bind(this) }>Updated {this.props.buildTime}</a>
-        </div>
```

Comment:
Do we want to kill the `onClickGitHub()` and `onClickRevision()` methods as well? [./server/src/views/frame.js:614-620](https://github.com/mozilla-services/pageshot/blob/901321a4ce477d160eef8c983efe8abb1f90f5a9/server/src/views/frame.js#L614-L620)